14_contains/solution/contains-solution.js

```diff
@@ -0,0 +1,16 @@
+const contains = function(obj, searchValue) {
+  const values = Object.values(obj);
+
+  // NaN !== NaN, so we would usually have to do an explicit check with Math.isNaN to test for it. However, Array.prototype.includes handles that for us
```

Comment:
```suggestion
  // NaN === NaN evaluates to false (as per the IEEE 754 standard)
  // Normally, we would have to do an explicit Number.isNaN() check to compare NaN equality
  // However, Array.prototype.includes automatically handles this for us
```
Nit: Personally, I think something like this is a little clearer as to what the problem is (NaN === NaN being false), what the usual resolution would be, and then how our solution works. What are your thoughts?

More importantly, it's `Number.isNaN` as opposed to `Math.isNaN`

Reply:
I think this is a good change, however I don't think it would be necessary to include the IEEE bit. If we *do* then it may be worth linking to the wiki page

Reply:
You could leave a simple link, though it won't be like markdown because it's a JS comment, so will just be a bare link.
The IEEE part isn't particularly necessary, more so the structure of the comments IMO

Reply:
Thanks!